### PR TITLE
Update changelog

### PR DESCRIPTION
**Pull request by demartinofra**
Signed-off-by: Francesco De Martino <fdm@amazon.com>

**Please See** [Git Pull Request Instructions](https://github.com/aws/aws-parallelcluster/wiki/Git-Pull-Request-Instructions)

*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,63 +5,85 @@ CHANGELOG
...
+- Remove support for SGE and Torque schedulers.
+- Remove support for CentOS8.
 - Change format and syntax of the configuration file to be used to create the cluster, from ini to YAML.
-- Use AWS Cloud Development Kit to define and generate CloudFormation templates. Add Node.js as requirement.
-- Rename all the commands, refactor parameters and use JSON as output format.
-- Remove interactivity from commands used to create, update, delete and describe a cluster.
+- Remove `--cluster-template`, `--extra-parameters` and `--tags` parameters for the `create` command.
+- Remove `--cluster-template`, `--extra-parameters`, `--reset-desired` and `--yes` parameters for the `update` command.
+- Remove `--config` parameter for `delete`, `status`, `start`, `stop`, `instances` and `list` commands.
+- Add `--suppress-validators`, `--validation-failure-level` and`--disable-update-check` parameters to `create` command.
```
Comment (mauri-melato) [inline CHANGELOG.md:32]: This `--disable-update-check` was not added

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,63 +5,85 @@ CHANGELOG
...
+- Add `--suppress-validators` and `--validation-failure-level` parameters to `update` command.
 - Remove possibility to specify aliases for `ssh` command in the configuration file.
-- Distribute AWS Batch commands: `awsbhosts`, `awsbkill`, `awsbout`, `awsbqueues`, `awsbstat` and `awsbsub`
-  as a separate `aws-parallelcluster-awsbatch-cli` PyPI package.
+- Rename `createami` command to `build-image` command, remove `--ami-id`, `--os`, `--instance-type`,
+  `--ami-name-prefix`, `--custom-cookbook`, `--post-install`, `--no-public-ip`, `--cluster-template`, `--vpc-id`,
+  `--subnet-id`.
+- Add `--image-name`, `--config`, `--region` parameters to `build-image` command.
+- Add `delete-image` command with `--name`, `--region`, `--force` parameters.
```
Comment (mauri-melato) [inline CHANGELOG.md:39]: New commands / options / functionalities should be moved  in the enhancements section. 
Also the new build image requires a sentence:
  Renew custom AMI creation and management that leverages EC2 Image Builder

File: CHANGELOG.md [hunk 3]
```diff
@@ -5,63 +5,85 @@ CHANGELOG
...
-  - Rename `NodeType` and tags from `Master` to `HeadNode`.
-- Remove `parallelcluster-` prefix from CloudFormation stack created by ParallelCluster.
+- Create a single ParallelCluster S3 bucket for each AWS region rather than for each cluster.
+- Adopt inclusive language
+  - Rename MasterServer to HeadNode in CLI outputs.
+  - Rename variable exported in the AWS Batch job environment from MASTER_IP to PCLUSTER_HEAD_NODE_IP.
+  - Rename all CFN outputs from Master* to HeadNode*.
+  - Rename NodeType and tags from Master to HeadNode.
+- Remove Ganglia support.
+- Add support for associating an existing Elastic IP to the head node.
```
Comment (mauri-melato) [inline CHANGELOG.md:60]: Enhancement

File: CHANGELOG.md [hunk 4]
```diff
@@ -5,63 +5,85 @@ CHANGELOG
...
+- Upgrade Cinc Client to version 17.2.29.
 - Upgrade Slurm to version 20.11.8.
+- Upgrade NICE DCV to version 2021.1-10851.
 - Upgrade Cinc Client to version 17.2.29.
-- CLI commands do not default to `~/.parallelcluster/config` file anymore. The cluster configuration is now a required parameter.
-- Slurm: Extend limits for supported number of queues to 10 and number of compute resources per queue to 5.
+- Upgrade NVIDIA driver to version 470.57.02.
+- Upgrade CUDA library to version 11.4.0.
+- Remove installation of MPICH and FFTW packages.
+- Upgrade EFA installer to version 1.13.0
```
Comment (mauri-melato) [inline CHANGELOG.md:84]: Let's do all the  Upgrades first then all the Removes
Upgrade list: EFA, Slurm, DCV, ...[the rest]

File: CHANGELOG.md [hunk 5]
```diff
@@ -5,63 +5,85 @@ CHANGELOG
...
-- Restrict access to Instance Metadata Service (IMDS) and add configuration parameter to manage it.
-- Remove instance store software encryption option (encrypted_ephemeral).
+- Remove [runtime creation method](https://docs.aws.amazon.com/parallelcluster/latest/ug/tutorials_02_ami_customization.html)
+  of custom ParallelCluster AMIs.
+- Retain CloudWatch logs on cluster deletion by default. If you want to delete the logs during cluster deletion, set
+  `Monitoring > Logs > CloudWatch > RetainOnDeletion` to False in the configuration file.
+- Remove instance store software encryption option (encrypted_ephemeral) and rely on default hardware encryption provided
+  by NVMe instance store volumes.
+- Add tag 'Name' to every shared storage with the value specified in the shared storage name config.
+- Slurm: Extend limits for supported number of queues to 10 and number of compute resources per queue to 5.
```
Comment (mauri-melato) [inline CHANGELOG.md:76]: Enhancement

File: CHANGELOG.md [hunk 6]
```diff
@@ -5,63 +5,85 @@ CHANGELOG
...
-- Remove `parallelcluster-` prefix from CloudFormation stack created by ParallelCluster.
+- Create a single ParallelCluster S3 bucket for each AWS region rather than for each cluster.
+- Adopt inclusive language
+  - Rename MasterServer to HeadNode in CLI outputs.
+  - Rename variable exported in the AWS Batch job environment from MASTER_IP to PCLUSTER_HEAD_NODE_IP.
+  - Rename all CFN outputs from Master* to HeadNode*.
+  - Rename NodeType and tags from Master to HeadNode.
+- Remove Ganglia support.
+- Add support for associating an existing Elastic IP to the head node.
+- Encrypt root EBS volumes and shared EBS volumes by default. Note that if the scheduler is AWS Batch, the root volumes
```
Comment (mauri-melato) [inline CHANGELOG.md:61]: Enhancement

File: CHANGELOG.md [hunk 7]
```diff
@@ -5,63 +5,85 @@ CHANGELOG
...
-  - Rename all CloudFormation outputs from `Master*` to `HeadNode*`.
-  - Rename `NodeType` and tags from `Master` to `HeadNode`.
-- Remove `parallelcluster-` prefix from CloudFormation stack created by ParallelCluster.
+- Create a single ParallelCluster S3 bucket for each AWS region rather than for each cluster.
+- Adopt inclusive language
+  - Rename MasterServer to HeadNode in CLI outputs.
+  - Rename variable exported in the AWS Batch job environment from MASTER_IP to PCLUSTER_HEAD_NODE_IP.
+  - Rename all CFN outputs from Master* to HeadNode*.
+  - Rename NodeType and tags from Master to HeadNode.
+- Remove Ganglia support.
```
Comment (mauri-melato) [inline CHANGELOG.md:59]: To the bottom

File: CHANGELOG.md [hunk 8]
```diff
@@ -5,63 +5,85 @@ CHANGELOG
...
+- Add configuration HeadNode.Imds.Secured to enable/disable restricted access to Instance Metadata Service (IMDS).
+- Implement scaling protection mechanism with Slurm scheduler: compute fleet is automatically set to 'PROTECTED' 
+  state in case recurrent failures are encountered when provisioning nodes.
 
 **CHANGES**
-- Drop support for SGE and Torque schedulers.
-- Drop support for CentOS8.
+- Remove support for SGE and Torque schedulers.
+- Remove support for CentOS8.
 - Change format and syntax of the configuration file to be used to create the cluster, from ini to YAML.
```
Comment (mauri-melato) [inline CHANGELOG.md:28]: Let's also add that a configuration file now represents the settings of a single cluster.

File: CHANGELOG.md [hunk 9]
```diff
@@ -5,63 +5,85 @@ CHANGELOG
...
-- Use different permissions in instance roles based on the scheduler and the node's role in the cluster.
-- Implement scaling protection mechanism with Slurm scheduler: compute fleet is automatically set to 'PROTECTED' state
-  in case recurrent failures are encountered when provisioning nodes.
-- Add tag `Name` to every shared storage with the value specified in the shared storage name config.
+- Add support for multiple subnets when using AWS Batch.
+- Allow full control on the name of CloudFormation stacks created by ParallelCluster by removing the `parallelcluster-`
+  prefix.
+- Add multiple queues and compute resources support for pcluster configure when the scheduler is Slurm.
+- Add prompt for availability zone in pcluster configure automated subnets creation.
+- Add configuration HeadNode.Imds.Secured to enable/disable restricted access to Instance Metadata Service (IMDS).
```
Comment (mauri-melato) [inline CHANGELOG.md:21]: For consistency I think anywhere else we used / notations, HeadNode / Imds / Secured ?

File: CHANGELOG.md [hunk 10]
```diff
@@ -5,63 +5,85 @@ CHANGELOG
...
-- Add possibility to use an existing Private Route53 Hosted Zone when using Slurm as scheduler.
-- Add possibility to suppress validators and filter validation failures by failure level at cluster creation and update time.
-- Add new set of commands to build and manage Custom AMIs.
-- Add new set of commands to list, view and export cluster and image builder logs.
-- Split head node and compute fleet instance roles and add possibility to configure a different instance role
-  for each queue.
+- Add support for pcluster actions (e.g., create-cluster, update-cluster, delete-cluster) through HTTP endpoints
+  with Amazon API Gateway.
+- Add support to use an existing Instance Profile for cluster creation and custom AMI creation.
+- Support restart/reboot for EC2 instance types with
```
Comment (mauri-melato) [inline CHANGELOG.md:11]: Enable the possibility to restart / reboot the head node also for instance types with instance store(...). Those operations remain anyway managed by the user that is responsible for the status of the cluster while operating on the head node.

**Review by mauri-melato — COMMENTED** [review]
Please address the changes below and reach me in case of doubts, thanks.

File: CHANGELOG.md [hunk 11]
```diff
@@ -5,63 +5,85 @@ CHANGELOG
...
-  per cluster.
-- Encrypt root EBS volumes and shared EBS volumes by default.
-  Note that if the scheduler is AWS Batch, the root volumes of the compute nodes cannot be encrypted by ParallelCluster.
-- Add multiple queues and compute resources support for `pcluster configure` when the scheduler is Slurm.
-- Add prompt for availability zone in `pcluster configure` automated subnets creation.
-- Use different permissions in instance roles based on the scheduler and the node's role in the cluster.
-- Implement scaling protection mechanism with Slurm scheduler: compute fleet is automatically set to 'PROTECTED' state
-  in case recurrent failures are encountered when provisioning nodes.
-- Add tag `Name` to every shared storage with the value specified in the shared storage name config.
+- Add support for multiple subnets when using AWS Batch.
```
Comment (mauri-melato) [inline CHANGELOG.md:16]: Remove

File: CHANGELOG.md (hunk 6 above)
Comment (demartinofra) [reply CHANGELOG.md:61]: humm isn't this more like a change?

Comment (mauri-melato) [reply CHANGELOG.md:61]: I'd sell it as an enhanced security posture

File: CHANGELOG.md [hunk 12]
```diff
@@ -5,63 +5,85 @@ CHANGELOG
...
+- Add the possibility to configure IAM role, profile and policies per head node and for each compute queue
 - Add possibility to configure different security groups for each queue.
+- Allow full control on the name of CloudFormation stacks created by ParallelCluster by removing the `parallelcluster-`
+  prefix.
+- Add multiple queues and compute resources support for pcluster configure when the scheduler is Slurm.
+- Add prompt for availability zone in pcluster configure automated subnets creation.
+- Add configuration `HeadNode / Imds / Secured` to enable/disable restricted access to Instance Metadata Service (IMDS).
+- Implement scaling protection mechanism with Slurm scheduler: compute fleet is automatically set to 'PROTECTED' 
+  state in case recurrent failures are encountered when provisioning nodes.
+- Add `--suppress-validators`, `--validation-failure-level` and`--disable-update-check` parameters to `create` command.
```
Comment (mauri-melato) [inline CHANGELOG.md:26]: Remove --disable-update-check  - it was not added

**Review by mauri-melato — COMMENTED** [review]
Only, few feedback remaining to be addressed, thanks!

**Review by mauri-melato — COMMENTED** [review]
---

**Review by lukeseawalker — APPROVED** [review]

**Pull request merged**